content/variables/using-environment-variables.md

```diff
@@ -29,6 +29,20 @@ Add-Content -Path $env:CM_ENV -Value "KEY=value"
 
 You can then reference the variable in subsequent parts of your workflow by using `$env:KEY`.
 
+### Setting a multiline environment variable
+
+To add a multiline environment variable, you need to use `<<` instead of an `=` to mark the end of the key in the key value pair. In addition, set a delimiter to mark the start and end of the variable.
```

Comment:
```suggestion
To add a multiline environment variable, you need to use `<<` instead of an `=` to mark the end of the key in the key-value pair. In addition, set a delimiter to mark the start and the end of the variable.
```
Also, add the requirements for the delimiter here.